SnowSQL/public/Invoke-SnowSql.ps1

```diff
@@ -44,6 +44,12 @@ function Invoke-SnowSql
         [string[]]
         $Query = '!help',
 
+        # Timeout query
+        [Parameter(ParameterSetName = 'QueryCred')]
+        [Parameter(ParameterSetName = 'QueryConnection')]
```

Comment:
Should this work for all parameters? If so, you can remove the `ParameterSetName` and use an empty `[Parameter()]` attribute.

Reply:
Being only an option for the connection, I had just applied it for these two parameters.
https://docs.snowflake.com/en/user-guide/snowsql-config.html#login-timeout